### DISCUSSION TITLE
search of stars in the zc catalogue

### DISCUSSION THREAD

**Discussion by gehrigu** [Stellarium/stellarium#2953] [Q&A]
Hi there

How can I search for a star if I only know the star number in the zc catalogue (Robertson's Zodiacal Catalogue, mainly used for lunar occultation observation).

Many thanks for your hints,
Urs

**Comment by gzotti** [comment]
It seems SIMBAD does not use it, so you are out of luck. Find an external cross-index to HIP numbers.
